CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 ### Deprecated
 ### Removed
+- Description prop in the [`NavBar`](http://localhost:8080/?selectedKind=NavBar&selectedStory=NavBar) component has been removed
```

Comment:
I think this should either be referencing our live documentation site or the live storybook. Probably the docs site?

Reply:
changed to docs link